images/airflow-managed-services/Dockerfile

```diff
@@ -22,7 +25,8 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
 RUN unzip awscliv2.zip
 RUN ./aws/install
 RUN curl -sL https://aka.ms/InstallAzureCLIDeb | bash
-COPY --from=build /osde2e/out/osde2e* /usr/local/bin/
+RUN cp /osde2e/out/osde2e* /usr/local/bin/
```

Comment:
Do we need this in here anymore? Rosa OVN was the only thing that was using osde2e that I can recall.

Reply:
Happy to nuke it. I just don't have enough context :)

Reply:
@morenod can you confirm we don't use osde2e for any of the other jobs (I'm 99% sure of that) and let's get it out of here its just taking up space.

Reply:
If there's other places where osde2e is referenced, there probably has to be a different PR to remove it though.

Reply:
osde2e is only used for OVN installation. I have reviewed that it is removed from everywhere on the PR

Reply:
Ack, thanks!